openid-4-verifiable-presentations-1_0.md

```diff
@@ -706,6 +724,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+This section provides guidance on processing and security considerations for the `attachments` parameter defined in (#new_parameters).
```

Comment:
I think a slightly longer introduction here would be helpful - e.g. in the same way as https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-8.4 does at least give an example of what it could be used for?

Reply:
Added some examples. As mentioned above the logic if something should be aborted or so is defined by the value of the format and therefore by the context.